### PR TITLE
Boar 2

### PR DESCRIPTION
**Pull request by Eugene44-hub**

### CODE REVIEW

File: src/components/Buttons/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const Button = (props:Props) => {
+  return (
+
+    <NativeBtn onPress={props.callback} width="100%" fontFamily="ReadexProBold" p={0} height={62} backgroundColor="accent_bg.50" borderRadius={12}>
+    <Text fontFamily="ReadexProBold" fontSize={16} color="white">{props.text}</Text>
+    </NativeBtn>
+
+  )
+}
+
```
Comment (jbrainz) [inline src/components/Buttons/Button.tsx:19]: you can destrucure out the props: ({callback, text}: Props) 🙂

File: src/components/Headers/FormHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Box, Container,Heading, Text, VStack } from "native-base";
+
+interface Props{
+text?:string;
+header:string
+}
+
+const FormHeader = (props:Props) => {
```
Comment (jbrainz) [inline src/components/Headers/FormHeader.tsx:9]: destructure props as suggested before 👆🏼

File: src/components/OnBoarding.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { Box, Center, Text, Circle, Image, Heading, HStack, Container } from "native-base";
+import { TouchableOpacity } from 'react-native';
+import Pagination from './Pagination';
+
+interface Props {
+  bgColor: string;
+  circleOneColor: string;
+  circleTwoColor: string;
+navigation:any;
+  image: any;
```
Comment (jbrainz) [inline src/components/OnBoarding.tsx:11]: please don't use any 🙂

File: src/components/OnBoarding.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  header: string;
+  text: string;
+  data:any[];
+  index:number;
+}
+
+const OnBoarding = (props: Props) => {
+
```
Comment (jbrainz) [inline src/components/OnBoarding.tsx:19]: you can define the structure of the data? rather that using any. 😭

File: src/components/Pagination.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { Box, Circle, HStack } from 'native-base'
+import React from 'react'
+
+interface Props{
+  data:any[];
```
Comment (jbrainz) [inline src/components/Pagination.tsx:5]: 😭😭😭

File: src/components/avatars/SexAvatar.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import { Box, Image, Text, Circle } from 'native-base'
+import React from 'react'
+
+interface Props {
+  sex: string;
+  image: any;
```
Comment (jbrainz) [inline src/components/avatars/SexAvatar.tsx:6]: `import { Image as ImageNative } from 'react-native';`
`source: ImageNative.propTypes.source.isRequired`
don't use any.

File: src/navigation/AuthNavigation.tsx [hunk 7]
```diff
@@ -2,14 +2,35 @@ import { createNativeStackNavigator } from "@react-navigation/native-stack";
...
+import LoginPin from "../screens/pin/LoginPin";
+import ResetPin from "../screens/pin/ResetPin";
+import TransactionPin from "../screens/pin/TransactionPin";
 const AuthNavigation = (): JSX.Element => {
   const Stack = createNativeStackNavigator<AuthRoutes>();
   return (
-      <Stack.Navigator initialRouteName="SignUp">
-        <Stack.Screen name="SignUp" component={SignUp} />
-        <Stack.Screen name="Login" component={SignIn}/>
+      <Stack.Navigator initialRouteName="Onboarding">
```
Comment (jbrainz) [inline src/navigation/AuthNavigation.tsx:20]:
```
 screenOptions={{
    headerShown: false
  }}
```
add this in `<Stack.Navigator `.... and remove the repetative setting in the stack screen.

File: src/screens/password/Phone.tsx [hunk 8]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/no-children-prop */
+import React from 'react'
+import { Box, Center, Input, InputGroup, InputLeftAddon, Text } from 'native-base'
+import Button from '../../components/Buttons/Button'
+import FormHeader from '../../components/Headers/FormHeader'
+import InputField from '../../components/InputField'
+import { AuthNavigationProps } from '../../types/routes'
+import {useSelector} from "react-redux"
+const Phone = ({navigation}:AuthNavigationProps<"Email">):JSX.Element => {
+const {user}=useSelector((state:any)=>state.user)
```
Comment (jbrainz) [inline src/screens/password/Phone.tsx:10]: any ?

**Review by jbrainz — CHANGES_REQUESTED** [review]
Please make this changes then, sqaush your commit so we can merge. 😊

File: src/components/Buttons/Button.tsx (hunk 1 above)
Comment (Eugene44-hub) [reply src/components/Buttons/Button.tsx:19]: done

File: src/components/Headers/FormHeader.tsx (hunk 2 above)
Comment (Eugene44-hub) [reply src/components/Headers/FormHeader.tsx:9]: done

File: src/components/OnBoarding.tsx (hunk 3 above)
Comment (Eugene44-hub) [reply src/components/OnBoarding.tsx:11]: done

File: src/components/OnBoarding.tsx (hunk 4 above)
Comment (Eugene44-hub) [reply src/components/OnBoarding.tsx:19]: done

File: src/components/Pagination.tsx (hunk 5 above)
Comment (Eugene44-hub) [reply src/components/Pagination.tsx:5]: done

**Pull request merged**